Projectile behavior

TheNathan27 opened the pull request:
I made the projectile move in a strait line while being affected by gravity. The projectile destroys itself when a timer is reached or it collides with the ground.

TheNathan27 committed:
Projectile behavior

I made the projectile move in a strait line while being affected by gravity. The projectile destroys itself when a timer is reached or it collides with the ground.

CSwithBaez commented:
Please rename the script and prefab as it conflicts with another bullet script. This behavior can be added to the current bullet script if necessary.

TheNathan27 committed:
Improved Bullet Behavior

The bullet will now delete itself no matter what it hits. To do this, I moved the fire point on the player prefab forward 1 unit.
